### PR TITLE
Fix typo in benchmark utils

### PR DESCRIPTION
**Pull request by quasiben**
@manopapad  it looked you branched from `main`.  I cherry picked the commit and applied it to branch-0.17 for a clean commit history 

Supersedes https://github.com/rapidsai/dask-cuda/pull/415/commits/

### CODE REVIEW

**Comment by pentschev** [comment]
Could you then do https://github.com/rapidsai/dask-cuda/pull/415#issuecomment-709320908 ?

**Comment by quasiben** [comment]
> Could you then do #415 (comment) ?

Addressed in 44b8543

**Comment by quasiben** [comment]
@pentschev when you have time can you give this another review ?

File: dask_cuda/benchmarks/utils.py [hunk 1]
```diff
@@ -71,12 +71,6 @@ def parse_benchmark_args(description="Generic dask-cuda Benchmark", args_list=[]
...
     )
-    parser.add_argument(
-        "--single-node",
-        action="store_true",
-        dest="multi_node",
-        help="Runs a single-node cluster on the current host.",
-    )
     parser.add_argument(
         "--multi-node",
         action="store_true",
```
Comment (pentschev) [inline dask_cuda/benchmarks/utils.py:76]:
```suggestion
        action="store_false",
```

**Review by pentschev — CHANGES_REQUESTED** [review]

**Comment by pentschev** [comment]
@quasiben we want `--multi-node` to be `False` by default, added a suggestion above.

**Comment by quasiben** [comment]
Dine

**Review by pentschev — APPROVED** [review]
Thanks @quasiben !

**Comment by quasiben** [comment]
Wait, when `--multi-node` is not set it should be `False` in which case, I think we want `store_true`, right ?

**Comment by pentschev** [comment]
> Wait, when `--multi-node` is not set it should be `False` in which case, I think we want `store_true`, right ?

Yes, you're right. I confused things with the whole `--enable-`/`--disable-` behavior. We do want to remove `--single-node` (and this is implicitly the default behavior), and have `--multi-node` set to `store_true`. Can you revert the latest commit?

**Comment by quasiben** [comment]
Thanks for the confirmation.  Reverted the commit

**Comment by pentschev** [comment]
Thanks @quasiben for noticing!

**Pull request merged**